src/EditorFeatures/CSharpTest/Intents/AddConstructorParameterIntentTests.cs

```diff
@@ -33,7 +33,17 @@ public async Task AddConstructorParameterWithField()
 {
     private readonly int _someInt;{|priorSelection:|}
 
-    public C({|typed:int som|})
+    public C()
```

Comment:
some test refactoring to make it clear what the 'prior to detection', 'current' and 'expected after application' text is supposed to be.  Before they were lumped together

Reply:
This is definitely fine and I don't feel strongly about it, but I actually like the way it looked before, as it has less text and easier to see the diff.

Reply:
Yeah it was definitely less code.  THe main problem was that it did not handle replacements very well (you can't specify what text to remove and add inline without nesting).  And if the change needs to span multiple distinct locations the annotations and the code to handle the annotations gets even more complicated.  I also forget what annotations to use each time I come back to this 😆 

IMHO the more verbose but explicit description of each step now makes the more complicated scenarios easier to reason about.